test/test_mdlp.py

```diff
@@ -1,16 +1,8 @@
-from sklearn.utils.testing import assert_array_equal
```

Comment:
Does `tests -> test` matter that much to you? Scikit-learn uses `tests`, and I'd prefer to keep this PR to as few changes as possible.

Reply:
sounds good.